src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -2485,3 +2486,37 @@ class FunctionSegment(ansi.FunctionSegment):
             ),
         ),
     )
+
+
+class CreateExternalFunctionStatementSegment(BaseSegment):
+    """A `CREATE EXTERNAL FUNCTION` segment.
+
+    https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_EXTERNAL_FUNCTION.html
+    """
+
+    type = "create_external_function_statement"
+    match_grammar = Sequence(
+        "CREATE",
+        Sequence("OR", "REPLACE", optional=True),
```

Comment:
Mind replacing the ~3 instances of this in `dialect_redshift` with `Ref("OrReplaceGrammar", optional=True),`?